src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/BundleEntityBuilder.java

```diff
@@ -13,6 +13,8 @@
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
 
+import java.util.Map;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Map'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=53&issues=AWU6Y48VWWqqPQVjPd1w&open=AWU6Y48VWWqqPQVjPd1w)

[ISSUE_KEY]: AWU6Y48VWWqqPQVjPd1w